programs/ssl/ssl_server2.c

```diff
@@ -1653,8 +1655,10 @@ int main( int argc, char *argv[] )
 #if !defined(_WIN32)
         if( received_sigterm )
         {
-            polarssl_printf( " interrupted by signal\n" );
-            ret = 0;
+            polarssl_printf( " interrupted by SIGTERM (in net_accept())\n" );
+            if( ret == POLARSSL_ERR_NET_ACCEPT_FAILED )
```

Comment:
Should this be `POLARSSL_ERR_NET_RECV_FAILED?`

Reply:
Yes, in 1.3 `net_accept()` already returns its specific code. (Confirmed by reading the source and running `ssl_server2` and sending it SIGTERM while it's waiting for a new connection.) The only difference is that `net_read()` didn't distinguish between "generic error" and "the context is invalid".

Reply:
OK